google_nest_sdm/google_nest.py

```diff
@@ -163,17 +163,17 @@ def CreateCreds(args: argparse.Namespace) -> Credentials:
                 "installed": {
                     "client_id": args.client_id,
                     "client_secret": args.client_secret,
-                    "redirect_uris": ["urn:ietf:wg:oauth:2.0:oob"],
                     "auth_uri": OAUTH2_AUTHORIZE_FORMAT.format(
                         project_id=args.project_id
                     ),
                     "token_uri": OAUTH2_TOKEN,
                 },
             }
+            print(client_config)
             app_flow = InstalledAppFlow.from_client_config(
                 client_config, scopes=SDM_SCOPES
             )
-            creds = app_flow.run_console()
+            creds = app_flow.run_local_server()
```

Comment:
So this is really just using web auth even though its in an installed app class?

Reply:
Not sure if you saw this question, but just wanted to confirm what this is actually doing.

Reply:
This is my understanding: OOB is not supported anymore.

Migration guide: https://developers.google.com/identity/protocols/oauth2/resources/oob-migration?hl=en

Reply:
Hi, I know about that, I was asking a lower level question about the library naming.

Reply:
oh, I see. Yes, this is using web auth even if it is a installed app. This is my understanding.